functions_n_objects_lab.js

```diff
@@ -0,0 +1,154 @@
+// # Mid-Unit-1 Review Lab
+
+// In this lab, we will review two key topics covered so far: Objects and Functions.  These are two vital topics for understanding JavaScript fundamentals, and will be used heavily as we build web apps in later units.
+
+
+// ## Functions
+
+// ## 1. Average of two numbers
+// const average = (num1, num2) => {
+//     return (num1 + num2) / 2
+// }
+// console.log(average(5, 8))
+// Write a function that takes in two numbers as input and returns their average
+
+// ```
+// Input: 4,6
+// Output: 5
+// ```
+
+// ## 2. Average of three numbers
+// const average = (num1, num2, num3) => {
+//     return (num1 + num2 + num3) /3
+// }
+// console.log(average(3, 7, 9))
+// Write a function that takes in three numbers as input and returns their average
+
+// ```
+// Input: 1,3,5
+// Output: 3
+// ```
+
+// ## 3. Average of array
+// const avgArr = (arr) => {
+//     let sum = 0
+//     for(let i = 0; i < arr.length; i++) {
+//         sum += arr[i]
+//     }
+//     return sum / arr.length
+// }
+// console.log(avgArr([7, 9, 5]))
+// Write a function that takes in an array of numbers as input and returns their average
+
+// ```
+// Input: [1,2,3,4,5,10,10]
+// Output: 5
+// ```
+
+// ## 4. Remove even numbers
+let num = 0;
+const remArr = (arr) => {
+    for(let i = 0; i < arr.length; i ++) {
+        if(arr[i] % 2 === 1) { 
```

Comment:
we want to RETURN a new array of only odd numbers, remember that console logging something and returning something are completely different